src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
-body {
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 100;
+$calendar-border: $calendar-size / 100;
+$calendar-padding: $calendar-size / 10;
+$calendar-font-size: 0.3 * $calendar-size;
+$calendar-animation-size: 0.2 * $calendar-size;
```

Comment:
Variables should be defined at the beginning of the file. Consider moving all variables to the top of the SCSS file for better visibility and organization.